### ISSUE TITLE
npm-groovy-lint - downgrade to 15.0.0

### ISSUE THREAD

**Issue by lutin-malin** [oxsecurity/megalinter#4414]
Hi megalinter team, 

we can't upgrade to verison 8.1+ because of below issues in the `npm-groovy-lint` linter

- [npm-groovy-lint/428](https://github.com/nvuillam/npm-groovy-lint/issues/428)
- [npm-groovy-lint/422](https://github.com/nvuillam/npm-groovy-lint/issues/422)

since version 8.1.0 (where linter `npm-groovy-lint` was bump to 15.0.2)

Linked issues are not fixed, and according to linked issues, remediation is to stick to version 15.0.0.
Could it be downgraded and sticked to 15.0.0 until fixed by the maintainers of this linter ?

Thanks in advance,

**Comment by nvuillam** [comment]
Hi @lutin-malin , i happen to also be the maintainer of npm-groovy-lint, but i have issues to find time :/

I'll try to fix it , otherwise i'll downgrade :)

**Comment by lutin-malin** [comment]
I fully understand the time issues ! no problem.
I'd then suggest to simply downgrade for now, pending to have the right amount of time to do it properly, not in any hurry.

(note that i really appreciate megalinter, seemless and so wide support without burdain)

thanks again !

**Comment by alexanderbazhenoff** [comment]
Indeed. The latest version of npm-groovy-lint brought some troubles with classes from jenkins shared `@Library` (see [related bug](https://github.com/super-linter/super-linter/issues/6371)). For this reason we have to keep several versions in our registry: MegaLinter 8.0.0 for groovу and 8.3.0 for everything else. Also flavors versions. If they don't fix it I'll have to nail 8.0.0 down forever :)

**Comment by lutin-malin** [comment]
must be kept open until final resolution

**Comment by lutin-malin** [comment]
@nvuillam would you mind to pin the npm-groovy-lint version to 15.0.0 for the time being ?
